recipes/openquake/meta.yaml

```diff
@@ -0,0 +1,290 @@
+{% set version = "2.8.0" %}
+
+package:
+  name: openquake.engine
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/o/openquake.engine/openquake.engine-{{ version }}.tar.gz
+  sha256: 55aed57bb65395e733106e5a1b2cb3fde9d8966f40f3b02167ccc9446d356473
```

Comment:
@mhearne-usgs I changed the URL to a more "human readable" format and used sha256 from pypi.org.